### ISSUE TITLE
Plugin ignoring "jetHome" option

### ISSUE THREAD

**Issue by sirsavary** [excelsior-oss/excelsior-jet-gradle-plugin#31]
I currently have the plugin set up and working but unfortunately, it is completely ignoring the `jetHome` option and I have had to resort to passing `-Djet.home` on the CLI which is less than ideal. I dug through the source a bit but was unable to find out why it wasn't working.

The path provided on the commandline and to the `jetHome` property was exactly the same, so I'm not sure why it wasn't working. Not sure if it was being ignored entirely or what.

Here is the relevant part of the gradle configuration:

```
excelsiorJet {
    mainClass = "com.virii.gameseer.supervisor.EntryKt"

    globalOptimizer = true
    packaging = "native-bundle"

    jetHome = "C:\\Users\\Jesse\\Development\\Tools\\excelsiorjet\\12-x86"
}
```

and the command flag that works:
```
-Djet.home="C:\Users\Jesse\Development\Tools\excelsiorjet\12-x86"
```

**Comment by pjBooms** [comment]
Thank you for the bug report.

It is a bug in the Gradle plugin that will be fixed in the next version.

**Comment by pjBooms** [comment]
The issue is fixed in 1.1.1 release. So please try and let us know about results.
